### DISCUSSION TITLE
Need to worry about log4j vulnerability in thanos?

### DISCUSSION THREAD

**Discussion by sreekanthm** [thanos-io/thanos#4966] [Questions & Answers]
We are using  “quay.io/thanos/thanos:v0.20.0” image in observability cluster and remote clusters. Do we need to worry about log4j vulnerability in this case, If yes, please guide us for the next action items

**Comment by GiedriusS** [comment]
Hi, no, Thanos is fine: https://github.com/thanos-io/thanos/issues/4951.

**Reply by sreekanthm** [reply]
Thanks for quick response, such a relief
